### ISSUE TITLE
Automatic releases on PyPi via Travis CI

### ISSUE THREAD

**Issue by DanHickstein** [PyAbel/PyAbel#163]
I made a PR to try to get Travis CI to automatically release each new version (not every PR, just when we click the "Draft a new release" button): https://github.com/PyAbel/PyAbel/pull/161

It seems pretty easy, as described here: https://docs.travis-ci.com/user/deployment/pypi

And it almost works. Travis attempts to deploy, but gets the following error (full output [here](https://travis-ci.org/PyAbel/PyAbel/jobs/123199540)):

```
Preparing deploy
Authenticated as DanHickstein

Deploying application
python: can't open file 'setup.py': [Errno 2] No such file or directory
ValueError: Cannot find file (or expand pattern): 'dist/*'
python: can't open file 'setup.py': [Errno 2] No such file or directory
```

I'll work more of this when I get a chance, but I thought that I'd open an Issue to explain the effort and see if anyone has ideas. Also, posted this on [SO](http://stackoverflow.com/questions/36635650/travisci-deploy-cannot-find-setup-py).

**Comment by DanHickstein** [comment]
Fixed this problem, then got stuck again. Turns out that Travis cannot handle certain symbols in a PyPi password.

Now I am victorious! Ha!

https://pypi.python.org/pypi/PyAbel

PyAbel will be automatically released each time we make a new release (or, any "tagged" commit, which I think basically only happens for a release). 

I bumped the version number to 0.7.4 in the midst of hacking this together.
